packages/block-library/src/block/edit.js

```diff
@@ -44,6 +44,11 @@ export default function ReusableBlockEdit( { attributes: { ref }, clientId } ) {
 		[ clientId ]
 	);
 
+	const innerBlockCount = useSelect(
+		( select ) => select( blockEditorStore ).getBlockCount( clientId ),
+		[ clientId ]
+	);
```

Comment:
Since we're already selecting data from the block editor store, I would recommend combining these selectors. There's nothing wrong with using multiple `useSelect` in components, but in this case, I think it makes sense to fetch data using a single selector.

Here's an example:

```js
const { canRemove, innerBlockCount } = useSelect(
  ( select ) => {
	  const { canRemoveBlock, getBlockCount } =
		  select( blockEditorStore );
  
	  return {
		  canRemove: canRemoveBlock( clientId ),
		  innerBlockCount: getBlockCount( clientId ),
	  };
  },
  [ clientId ]
);
```

Reply:
@Mamaduka agreed. Will combine them in the other files as well if there is an per-existing select.